beacon_node/network/src/sync/block_lookups/common.rs

```diff
@@ -174,9 +174,13 @@ impl<T: BeaconChainTypes> RequestState<T> for CustodyRequestState<T::EthSpec> {
         id: Id,
         // TODO(das): consider selecting peers that have custody but are in this set
         _peer_id: PeerId,
-        _: usize,
+        expected_blobs: usize,
         cx: &mut SyncNetworkContext<T>,
     ) -> Result<LookupRequestResult, LookupRequestError> {
+        if expected_blobs == 0 {
+            return Ok(LookupRequestResult::NoRequestNeeded("no columns to fetch"));
+        }
```

Comment:
Maybe move this into `custody_lookup_request` to group all LookupRequestResult conditions

Reply:
Done in https://github.com/sigp/lighthouse/pull/6814/commits/e6efd356873faed5f6b1f8f4f75597154c2cb04b